### PR TITLE
Whole genome seq data demo

### PR DESCRIPTION
**Pull request by jaybee84**
requesting review from @sgosline and @allaway

### CODE REVIEW

**Comment by jaybee84** [comment]
Output lives here: syn20370769

File: R_demos/3-wgseq-introduction.Rmd [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+```
+&nbsp;
+
+## Accessing or downloading data from Synapse
+
+Please look up the SynID of the relevant datafile and access it using the __synGet__ command of *synapser*. Load the required csv file containing the whole genome sequencing data directly into your Rstudio IDE by adding __$path__ to the accessed file-entity. 
+
+```{r load data, eval=TRUE, results='show', message=FALSE, warning=FALSE}
+
+#here() # check working directory
```
Comment (allaway) [inline R_demos/3-wgseq-introduction.Rmd]: suggest deleting this line, wd should be irrelevant here, right?

**Review by allaway — APPROVED** [review]
I requested one change that is not super important, so feel free to merge. We can review all of the mds  in a couple weeks in person to finalize them.

File: R_demos/3-wgseq-introduction.Rmd (hunk 1 above)
Comment (jaybee84) [reply R_demos/3-wgseq-introduction.Rmd]: Yeah..makes sense. Removed the line in the latest PR

**Pull request merged**